### DISCUSSION TITLE
How to invalidate a token when it is accidentally leaked

### DISCUSSION THREAD

**Discussion by HowieHz** [umami-software/umami#2696] [Q&A] [answered]
I used the method in the official documentation to get the token, but accidentally leaked my admin account token, resulting in someone else being able to access my umami admin account at will. I tried changing the admin account password, however that token is still valid.

**Comment by HowieHz** [comment]
@mikecao I'm sorry to bother you, but I was hoping you could take a look at this for me.

**Comment by HowieHz** [comment, marked as answer]
Looking at the source code, I see that modifying the hash salt invalidates the original token, but I'm not sure if this affects existing data in databases.

**Reply by mikecao** [reply]
This is correct. It won't affect the existing data.

**Reply by HowieHz** [reply]
Thank you for your reply. ❤️
